backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/externalsystem/invokeExternalSystem.feature

```diff
@@ -41,3 +41,16 @@ Feature: external system invocation using metasfresh api
       | param.key | param.value |
       | OrderId   | 111         |
       | OrderNo   | 222         |
+
+
+  @from:cucumber
+  Scenario: The request is good and the external Other is invoked via the correct process
+    Given add Other external system config with identifier: otherConfigIdentifier
+      | Name | Value |
+      | Test | Test  |
+
+    And store endpointPath /api/externalsystem/Other/:otherConfigIdentifier/test in context, resolving placeholder as ExternalSystem.value
+
+    When a 'POST' request is sent to metasfresh REST-API with endpointPath from context and fulfills with '200' status code
```

Comment:
also, please send some parameters and verify that those params make it to the external-systems invocation - together this the ones from the config

Reply:
just to be sure we are on the same page here:
  - the parameters we can send to the invoke ES rest-endpoint (i.e. `de.metas.common.externalsystem.JsonInvokeExternalSystemParams` ) are about AD_Process_Para, i.e. parameters used during the java process which do not necessarily end up in `de.metas.common.externalsystem.JsonExternalSystemRequest#parameters` (i.e. parameters sent over to the external systems)... so what we could verify is:
    - `ExternalSystem_Other_ConfigParameter` records we add will end up in `de.metas.common.externalsystem.JsonExternalSystemRequest#parameters`
    - `de.metas.common.externalsystem.JsonInvokeExternalSystemParams` we send will end up in `AD_PInstance_Para`